flink-runtime/src/main/scala/org/apache/flink/runtime/jobmanager/JobManager.scala

```diff
@@ -1891,8 +1891,8 @@ object JobManager {
     // parsing the command line arguments
     val (configuration: Configuration,
          executionMode: JobManagerMode,
-         listeningHost: String,
-         listeningPortRange: java.util.Iterator[Integer]) =
+         reachableHost: String,
```

Comment:
What does "reachableHost" mean here? I understand that it is not really the "listeningHost" any more (that is the wildcard address), but I don't understand what "reachableHost" should mean.